src/main/java/seedu/address/model/customer/Customer.java

```diff
@@ -27,7 +27,7 @@
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
+    public Customer(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
         requireAllNonNull(name, phone, email, address, tags);
```

Comment:
Looks good! I was thinking instead of using IDs of customer & product for transaction, we can check whether the customer.equals(otherCustomer) and product.equals(otherProduct), so we don't need an ID field